sql/CMakeLists.txt

```diff
@@ -543,6 +543,11 @@ ENDIF(WIN32)
 IF(NOT WITH_WSREP)
   SET(EXCL_WSREP "wsrep_[a-np-z]*.h")
 ENDIF()
+INSTALL(FILES
```

Comment:
Note, that for in-source case, the same files are installed twice: here, and couple of lines below.
It has never been a problem so far, but you need be aware. if you want exceptional future-bug-safety, you might add a guard  `IF( NOT ("${CMAKE_CURRENT_SOURCE_DIR}" STREQUAL ${CMAKE_CURRENT_BINARY_DIR}") ) ` or something like that.